### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by MrNaif2018** [pooler/electrum-ltc#306]
Hi @pooler! Is the project still maintained? It would be great if you could continue to update it with the latest electrum versions!

**Comment by pooler** [comment]
Unfortunately I haven't had the time to keep Electrum-LTC in sync with upstream lately. I honestly do not know when or if I'll be able to get back to it. I notified the Litecoin Foundation some time ago, and if everything goes well we will probably have a new maintainer soon.

**Comment by MrNaif2018** [comment]
Thank you very much. It was getting hard to keep electrum-ltc updated for BitcartCC needs when we upgraded electrums. I hope the situation gets better

**Comment by ghost** [comment]
@pooler 
[I verified that the fingerprints are correct](https://electrum-ltc.org/verifying-signatures.html) but...

*Loshan T (A.K.A. losh11) < loshan1212@gmail.com >* says different expiration date for keys:

```
pub   rsa4096/0x3B2A6315CD51A673 2016-01-09 [SC] [expires: 2028-05-06]
      Key fingerprint = C092 1846 FED0 BF4C F28B  E1D7 3B2A 6315 CD51 A673
uid                   [ unknown] Loshan T (A.K.A. losh11) <loshan1212@gmail.com>
sub   rsa4096/0xF913FB152B18335D 2016-01-09 [E] [expires: 2028-05-06]
      Key fingerprint = 9AC0 B4EF B6C3 47FC B954  5A02 F913 FB15 2B18 335D
```

**2028-05-06** not *2026-01-09* like the site instructions say?

This may not matter for me cause i'm using linux and the appimage is signed by your key. But macOS and Windows versions are signed by Loshan.

**Comment by pooler** [comment]
@Miss-Underst00d GPG keys can be renewed (and if necessary, revoked) by their owner. As long as you get the "Good signature" message when verifying the signature, it should be all good.

**Comment by MrCybertux** [comment]
Hi, @pooler is there any news on the new Maintainer?
Electrum was the only real desktop option in may option since i distrust company made multi coin wallets like exodus or atomic.
But the last release is over 14 Months old by now and MWEB is still not supported sadly.
I know non of this is your fault and i am very very thank full for the work you have already done. 🙂
But an update on the maintainer issue would be very welcome.

**Comment by pooler** [comment]
> Hi, @pooler is there any news on the new Maintainer?

My understanding is that @losh11 (developer at the Litecoin Foundation) has started looking into it.

> But the last release is over 14 Months old by now and MWEB is still not supported sadly.

Given the complexity of MWEB, I don't think we can realistically expect it to be implemented in Electrum any time soon.

**Comment by MrNaif2018** [comment]
I would recommend the new maintainer (if that's @losh11 or whoever it will be) to start from scratch. Current fork has lots of changes and each time merging there are lots of merge conflicts. I think what could be investigated is not renaming the electrum directory to electrum_ltc, but instead using the package_dir option in setup.py, e.g. https://github.com/bitcart/electrum-ltc/blob/7f42fc399dfc06b6245fcdfbb271ffa40df13daa/setup.py#L66-L68 (note: I have to update electrum-ltc from time to time to keep ltc supported in bitcart, but someone from litecoin foundation indeed needs to do it)

**Comment by losh11** [comment]
Yep, that’s the plan.

**Comment by losh11** [comment]
FYI: I've been making commits in this repo: https://github.com/litecoin-foundation/electrum-ltc
Got lots more locally, just trying to seperate out all the work into smaller commits, for future merge sake.

**Comment by MrCybertux** [comment]
@losh11 Any Updates on when there will be an new release? it has been almost 2 Years without updates

**Comment by debilin** [comment]
Any news?

**Comment by MrCybertux** [comment]
@debilin Slowly moving it seems @losh11 seems to be active on the LTC development but Electrum does not seem to be one of the main targets if looking on his github activity from August 2023 to now.
But he said he has made local changes that are not pushed to github yet we will see.

**Comment by losh11** [comment]
@MrCybertux this is accurate. After I'm finished with a big project I'm working on - I will get electrum synced again and get hector chu's mweb changed added on.

**Comment by MrCybertux** [comment]
@losh11 Thanks for your work! 👍️
Woop Woop Party time this is the feature i have been waiting for 🥳

**Comment by Panuchi** [comment]
We are waiting you🙏🙏🙏

**Comment by ghost** [comment]
I hope for good vibes and things to come. 
I don't know of any other Litecoin wallets on linux that are worth using so...

**Comment by Panuchi** [comment]
Still waiting @losh11 commits 🙏

**Comment by MrCybertux** [comment]
@losh11 is this app still getting released?
Sorry to say, but it is a finance/Money app that has not been updated for 3 Years and by now and that seems off.
We all try to be patient, but most of us are probably worried about the security.
Can you give us a little status update? 🙂


For everyone else there seems to be another LTC electrum fork or version here:
https://github.com/ltcmweb/electrum-ltc

**Comment by MrNaif2018** [comment]
The most maintained electrum-ltc fork is bitcart's one here:
https://github.com/bitcart/electrum-ltc
But I don't do binary releases, this is used by bitcart for LTC payment processing, though I do launch the gui to use it sometimes as well (:

Unfortunately the LTC foundation doesn't seem to be interested in maintaining an electrum which is a sad thing because electrum wallets are the best and more secure ones of it's kind
I've checked the commit history, and a change once in a year with some updates to the readme file only doesn't seem like progress...
This isn't hard but just requires a bit of time, and I hate that I have to update electrum-ltc instead of actual LTC folks, but well, it is what it is...

**Comment by losh11** [comment]
There is an upcoming update I am working on. We finally released Nexus Wallet for mobile devices, so now I have more capacity to work on Electrum-LTC. 

I have a newer branch, however I am currently testing this branch against the different hardware wallets. 

@MrCybertux this is no known security issue that exists in upstream.

@MrNaif2018 That's not true. And I would strongly suggest against using bitcart's fork of Electrum. It's untested, and if you use it, it's likely it'll perform some sort of migration that would be impossible to migrate to any future official release of 'Electrum-LTC'.

**Comment by MrNaif2018** [comment]
That is so funny to read, considering that I  update electrum-ltc against electrum upstream, which is exactly what pooler was doing for years and it worked perfectly 
Without updating with upstream you loose many of the features original developers introduce
And making the forks diverge is a big mistake
Also thanks for disregarding my work I've done for years to even maintain it, like I've told electrum-ltc isn't a big priority I guess 

P.S. I suggest checking what bitcart does and why it integrates with electrums, that's what allows people to accept LTC as well non-custodially for all those years, and it works only if you update electrum with upstream from time to time

**Comment by losh11** [comment]
@MrNaif2018 you completely misunderstood what I said. If people use your fork of Electrum-LTC, which causes a change to a users wallet, and this user switches back to a new release by mean, then some people are going to be really pissed off.

**Comment by MrNaif2018** [comment]
Yep, but naturally there would be no incompatibility with wallet formats if we keep to update with electrum upstream from time to time, like pooler did for all this time. Do you see any reason not to do that? I've seen attempts of other coins' wallets to do so, and usually it didn't work well

**Comment by VA1DER** [comment]
@losh11 Criticizing a (mostly) working release because a) it may make wallet changes consistent with upstream and b) it's untested when what you are offering is not even untested it's unobtanium... is puzzling.

@MrNaif2018 Thyank-you for the update.  I've sort of got it working but...

@everyone Please, please, please, **PLEASE**, for sanity's sake, make binary releases!  I know that devs LOVE pip, because it makes it easy for you, but it is the bane for people who have centrally managed Python installations.  It is also a nightmare for managing supply chain malware.  This is NOT the project to tell people to use pip for.  I may decide to trust you as a developer, but I don't want to have to trust multiple layers in a supply chain I don't manage.  Make a release.  For any uncommon dependencies (ones not available to a typical Debian/Ubuntu/Mint  system) include the modules you trust.  Otherwise, it's not a public-use project.

@MrNaif2018 I was able to get yours working by virtue of copying the dependent modules from an upstream Electrum binary release.  But this is FAR from ideal, and itself introduced quirks.

**Comment by MrNaif2018** [comment]
To make a release you can do it from the contrib folder, recent electrums have decent readme
No need to copy from other electrum binary releases

**Comment by hectorchu** [comment]
This exists now: https://ltc-electrum.github.io

**Comment by MrNaif2018** [comment]
@hectorchu Nice! Do you plan do update it with upstream occasionally? I might be able to switch from my long-maintained fork to yours in Bitcart finally.
Also it seems to support mweb as well?

**Comment by hectorchu** [comment]
@MrNaif2018 yes it shouldn't be too hard to merge upstream from time to time.
And yep it supports mweb.

**Comment by addy005** [comment]
> https://ltc-electrum.github.io

is this legit?

**Comment by addy005** [comment]
> This exists now: https://ltc-electrum.github.io

The Electrum LTC wallet, which was last updated 3–4 years ago, had many problems like crashes, slow syncing, “interface gave up” errors, and more. It was slow and buggy.

I saw this version, and it looks similar to Electrum LTC, so I’m not sure if it will be any faster or not. Also, can you tell me who manages it and where I can find the repository? Are you sure there are no backdoors in it?

I read that MWEB is used to hide balances and other details in Litecoin, but it’s taking too much time to sync. It should have an option to turn that on or off. I want to use it, but I also want to make sure it’s safe.

**Comment by hectorchu** [comment]
I made it, and no there are no backdoors. If I wanted to rug people there are easier ways than porting upstream master from scratch and adding MWEB capability.

**Comment by deutrino** [comment]
Please archive this project, the state of play is very confusing without doing a whole research project to determine whether @hectorchu's fork is legit - and I note there is no information about the possible confusion on the nearly-identical Github Pages site for that project, either 😐

**Comment by hectorchu** [comment]
> Please archive this project, the state of play is very confusing without doing a whole research project to determine whether [@hectorchu](https://github.com/hectorchu)'s fork is legit - and I note there is no information about the possible confusion on the nearly-identical Github Pages site for that project, either 😐

Why would my fork be any less legit than the one that hasn't been updated in 4 years and counting?
